docs/model_monitoring/index.md

```diff
@@ -1,27 +1,39 @@
 (model-monitoring)=
 
-# Model and data monitoring (beta)
+# Model, Data, and App monitoring
 
 ```{note}
-Model monitoring is based on Iguazio's streaming technology. Contact Iguazio to enable this feature.
+Monitoring required at the moment Iguazio's streaming technology. Open-source integration with Kafka is under development.
 ```
 
-MLRun's model monitoring service tracks the performance of models in production to help identify
-potential issues with concept drift and prediction accuracy before they impact business goals.
-Typically, model monitoring is used by devops for tracking model performance, and by data scientists to track model drift.
-Two monitoring types are supported:
-- Model operational performance (latency, requests per second, etc.).
-- Drift detection &mdash; identifies potential issues with the model. See [Drift Analysis](./model-monitoring-deployment.html#drift-analysis) for more details.
+```{note}
+This is currently a beta feature.
+```
+
+The MLRun's model monitoring service includes built-in model monitoring and reporting capability. With monitoring you get
+out-of-the-box analysis of:
 
-Model monitoring provides warning alerts that can be sent to stakeholders for processing.
+- **Model performance**: machine learning models train on data. It is important you know how well they perform in production.
+  When you analyze the model performance, it is important you monitor not just the overall model performance, but also the
+  feature-level performance. This gives you better insights for the reasons behind a particular result
+- **Data drift**: the change in model input data that potentially leads to model performance degradation. There are various
+  statistical metrics and drift metrics that you can use in order to identify data drift.
+- **Concept drift**: applies to the target sometimes the statistical properties of the target variable, which the model is
```

Comment:
Do you mean?
applies to the target. Sometimes the statistical properties of the target variable, which the model is
  trying to predict, change over time in unforeseen ways.